R/derive_joined.R

```diff
@@ -241,24 +270,54 @@
 #'   dataset_add = windows,
 #'   join_type = "all",
 #'   filter_join = AWLO <= ADY & ADY <= AWHI
-#' )
+#' ) %>%
+#'   select(USUBJID, ADY, AWLO, AWHI, AVISIT)
+#'
+#' @caption Join only the lowest/highest value occurring within a condition (`filter_join`,
+#'   `order` and `mode`)
+#' @info Derive the nadir value for each observation (i.e. the lowest value
```

Comment:
```suggestion
#' @info Derive the lowest value for each observation (i.e. the lowest value
```
nadir too fancy for a pleb like me

Reply:
Will leave this as its quite common wording in onco studies in particular, and the brackets explain what is meant here anyway for anyone who never heard the term. Plus it was in the examples already - i didn't add that, i.e. it wasn't me who went to fancy college! 😆